src/sass/partials/components/_basic-card.scss

```diff
@@ -1,5 +1,6 @@
 .basic-card {
   border: setting-get('input border width') solid setting-get('secondary color');
+  padding: setting-get('rhythm') / .5;
```

Comment:
* As we've discussed, don't divide by a decimal when you should be multiplying instead. This should be multiplied by 2
* Do we not have a double rhythm setting already? If not, we should